include/deal.II/sundials/ida.h

```diff
@@ -707,6 +707,15 @@ namespace SUNDIALS
     std::function<int(const VectorType &rhs, VectorType &dst)>
       solve_jacobian_system;
 
+#  if DEAL_II_SUNDIALS_VERSION_GTE(4, 0, 0)
+    /**
+     * A wrapper for a SUNLinearSolver.
+     */
+    LinearSolveFunction<VectorType> solve_jacobian_system_sundials;
+    std::unique_ptr<internal::LinearSolverWrapper<VectorType>>
+      jacobian_linear_solver;
```

Comment:
This is in the public interface. Can you explain in more detail what this function is supposed to do? It is also referencing an `internal::` class, which looks awkward in the public interface...

Reply:
That was a leftover...sorry ;).